run.sh

```diff
@@ -402,8 +392,6 @@ copyOrExtractMetaInformation() {
 
   unset CATEGORY_GENERATION_ONLY_ARR
   declare -g -A CATEGORY_GENERATION_ONLY_ARR
-  unset CATEGORY_MINIMAL_ARR
-  declare -g -A CATEGORY_MINIMAL_ARR
   fillTagArrayFromLine "$CATEGORY_GENERATION_ONLY"
   fillTagArrayFromLine "$CATEGORY_MINIMAL"
```

Comment:
This will break the script due to https://github.com/kyounger/casc-plugin-dependency-calculation/pull/32/files#diff-d31ce0453051853c17ba2a5225b3d1bfab548e095bab0967d6acfd1b3ce1b35dR329 but we don't have a test for using "gen only" plugins.

Reply:
We do now though :smile: